crates/proto/src/serialize/txt/rdata_parsers/svcb.rs

```diff
@@ -95,7 +95,12 @@ pub(crate) fn parse<'i, I: Iterator<Item = &'i str>>(mut tokens: I) -> ParseResu
         })?;
 
         // get the value, and remove any quotes
-        let value = key_value.next();
+        let mut value = key_value.next();
```

Comment:
I thought `trim_matches` was inappropriate because it _repeatedly_  removes matching prefixes/suffixes. I looked at a few of the other available stdlib fns and couldn't find one that did exactly what I wanted.

Reply:
Okay, fair. BTW, I don't think you have to escape `"` here, since it's in single quotes.

Reply:
Fixed to avoid the escaping. Good catch.